google-cloud-logging/README.md

```diff
@@ -44,48 +59,102 @@ entry.resource.labels[:version_id] = "20150925t173233"
 logging.write_entries entry
 ```
 
-## Rails Integration
+## Using Stackdriver Logging in frameworks
+
+The `google-cloud-logging` library provides framework integration for popular 
+Rack-based frameworks, such as Ruby on Rails and Sinatra, which sets the default 
+Rack logger to an instance of Stackdriver Logging logger.
```

Comment:
"instance of the Stackdriver Logging logger"

Reply:
Updated